packages/runtime/src/client/tests/client.get-component-snapshot.test.ts

```diff
@@ -15,7 +15,15 @@ function createTestClient() {
   return new MakeswiftClient(TEST_API_KEY, { runtime, apiOrigin })
 }
 
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
 describe('getComponentSnapshot using v1 element tree endpoint', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error').mockImplementation(() => {})
+  })
```

Comment:
I should have put this in a separate PR, this suppresses a `console.error` message in the `getComponentSnapshot` test output.